workflows/ampliseq.nf

```diff
@@ -117,9 +129,10 @@ multiqc_options.args       += params.multiqc_title ? " --title \"$params.multiqc
 
 def dada2_taxonomy_options  = modules['dada2_taxonomy']
 dada2_taxonomy_options.args += params.pacbio ? ", tryRC = TRUE" : ""
+dada2_taxonomy_options.args += params.iontorrent ? ", tryRC = TRUE" : ""
 
 def dada2_addspecies_options  = modules['dada2_addspecies']
-dada2_addspecies_options.args += params.pacbio ? ", tryRC = TRUE" : ""
+dada2_addspecies_options.args += params.iontorrent ? ", tryRC = TRUE" : ""
```

Comment:
Why are you removing `tryRC = TRUE` for `pacbio`? `tryRC = TRUE` is used for `pacbio` and `iontorrent` in `dada2_taxonomy_options`, why only for `iontorrent` but not `pacbio` in `dada2_addspecies_options`?

Reply:
That was not intentional, thanks for spotting it!